frontend/src/components/DashboardTable/DashboardTable.tsx

```diff
@@ -148,21 +148,37 @@ export default function DataTable() {
         <Alert
           onClose={() => setIsSnackbarOpen(false)}
           severity={successMessage ? 'success' : 'warning'}
-          variant="filled"
+          variant={successMessage ? 'filled' : 'standard'}
           sx={{ width: '100%' }}
         >
           {successMessage ? (
             <Typography>{successMessage}</Typography>
           ) : (
             <Box sx={{ display: 'flex', flexDirection: 'column' }}>
               <Typography>Are you sure you want to delete?</Typography>
-              <Button
-                color="inherit"
-                onClick={handleDelete}
-                sx={{ alignSelf: 'center' }}
+              <Box
+                sx={{
+                  display: 'flex',
+                  gap: '20px',
+                  alignItems: 'center',
+                  justifyContent: 'center',
+                }}
               >
-                Delete
-              </Button>
+                <Button
+                  color="primary"
+                  onClick={() => setIsSnackbarOpen(false)}
+                  sx={{ alignSelf: 'center' }}
+                >
+                  Cancel
+                </Button>
+                <Button
+                  color="inherit"
+                  onClick={handleDelete}
```

Comment:
delete button does not work. It shows the error: DELETE http://localhost:5001/api/v1/scholarships/6 422 (Unprocessable Content)